conf/auth.conf

```diff
@@ -57,6 +57,27 @@
 # (ie exactly as if auth yes was present).
 #
 
+# CONTROLLING FILE ACCESS (previously in fileserver.conf)
+
+# In previous versions of Puppet, you controlled file access by adding
+# rules to fileserver.conf. In Puppet 5 with Puppet Server, you can control
+# file access in auth.conf by controlling the /file_metadata(s)/<mount point>
+# and /file_content(s)/<mount point> paths. See the Puppet Server
+# documentation at
+# https://docs.puppet.com/puppetserver/latest/config_file_auth.html.
```

Comment:
Should we call out `static_file_content` too? Or were you thinking that's covered by the `/file_content/` regex since it doesn't have a start anchor?